src/anemoi/graphs/create.py

```diff
@@ -14,26 +18,9 @@ class GraphCreator:
 
     def __init__(
         self,
-        path,
-        config=None,
-        cache=None,
-        print=print,
-        overwrite=False,
-        **kwargs,
+        config: Union[PurePath, DotDict],
```

Comment:
nit: Thought about this more over the morning, I think this should actually be `Union[Path, DotDict]`.

Reply:
My thoughts exactly